### DISCUSSION TITLE
Does this Video DownloadHelper support to download from Netflix?

### DISCUSSION THREAD

**Discussion by Hagakui90** [aclap-dev/video-downloadhelper#735] [Help and Q&A]
### Your question

Does this Video DownloadHelper support to download from Netflix?

### Addon & CoApp version and details

![2024-05-28 11_45_41-Window](https://github.com/aclap-dev/video-downloadhelper/assets/109181861/71403970-7481-4cc3-815d-b0eb33ad5821)

**Comment by jcfrog** [comment]
hi,
no, Netflix uses DRM, so no.
